invoices/views.py

```diff
@@ -137,5 +139,14 @@ def delete_invoice(request, pk):
         return render(request, 'invoices/delete.html', {"invoice": invoice})
     if request.method == 'POST':
         invoice.delete()
+        receiver_name = invoice.owner.user.first_name + ' ' + invoice.owner.user.last_name
+        deleter_name = request.user.first_name + ' ' + request.user.last_name
+        requests.post("https://spam.datasektionen.se/api/sendmail", json={
```

Comment:
If I'm not wrong, this will always send an email about the invoice being deleted. Even if you delete your own invoice. Overkill? 

I would suggest we don't do this if `request.user` is the same as `invoice.owner.user`.